aws/resource_aws_wafregional_web_acl_test.go

```diff
@@ -223,6 +229,26 @@ func TestAccAWSWafRegionalWebAcl_changeRules(t *testing.T) {
 	})
 }
 
+// Calculates the index which isn't static because ruleId is generated as part of the test
+func computeWafRegionalWebAclRuleIndex(ruleId **string, priority int, action string, idx *int) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		ruleResource := resourceAwsWafRegionalWebAcl().Schema["rule"].Elem.(*schema.Resource)
+		actionType := map[string]interface{}{
+			"type": action,
+		}
+		m := map[string]interface{}{
+			"rule_id":  **ruleId,
+			"priority": priority,
+			"action":   actionType,
```

Comment:
@radeksimko I'm afraid I'm stuck here, can't figure out how to solve the puzzle how to pass the action in order to get the index.

Wanted to add some more tests steps for the rule fields, but this is blocking me.

Reply:
You can check by running the test `TestAccAWSWafRegionalWebAcl_changeRules`